diff --git a/aws_secretsmanager_agent/src/main.rs b/aws_secretsmanager_agent/src/main.rs
index 1eae01d..3bef3cf 100644
--- a/aws_secretsmanager_agent/src/main.rs
+++ b/aws_secretsmanager_agent/src/main.rs
@@ -174,7 +174,7 @@ async fn init(args: impl IntoIterator<Item = String>) -> (Config, TcpListener) {
     }
 
     // Bind the listener to the specified port
-    let addr: SocketAddr = ([127, 0, 0, 1], config.http_port()).into();
+    let addr: SocketAddr = ([0, 0, 0, 0], config.http_port()).into();
     let listener: TcpListener = TcpListener::bind(addr).await.unwrap_or_else(|err| {
         let msg = format!("Could not bind to {addr}: {}", err);
         error!("{msg}");
