diff --git a/aws_secretsmanager_agent/src/server.rs b/aws_secretsmanager_agent/src/server.rs
index e74968d..2e16d27 100644
--- a/aws_secretsmanager_agent/src/server.rs
+++ b/aws_secretsmanager_agent/src/server.rs
@@ -74,7 +74,6 @@ impl Server {
     /// * `std::io::Error` - Error while accepting request.
     pub async fn serve_request(&self) -> Result<(), Box<dyn std::error::Error>> {
         let (stream, _) = self.listener.accept().await?;
-        stream.set_ttl(1)?; // Prohibit network hops
         let io = TokioIo::new(stream);
         let svr_clone = self.clone();
         let rq_cnt = Arc::strong_count(&self.cache_mgr); // concurrent request count
